app/templates/main/download.html

```diff
@@ -72,23 +72,25 @@ <h1 class="govuk-heading-l">Download monitoring and evaluation data</h1>
                 "html": checkboxItems(outcomes["name"], outcomes["items"])
               }
             },
-            {
-              "heading": {
-                "text": "Filter by returns period"
-              },
-              "content": {
-                "html": selectItems(returnsParams)
-              }
-            }
           ]
         })
       }}
 
+<!-- # TODO: add back to accordian when returns period logic is implemented-->
+<!--            {-->
+<!--              "heading": {-->
+<!--                "text": "Filter by returns period"-->
+<!--              },-->
+<!--              "content": {-->
+<!--                "html": selectItems(returnsParams)-->
+<!--              }-->
+<!--            }-->
```

Comment:
@cyrusdobbs Can we use Jinja comments instead of HTML e.g. https://jinja.palletsprojects.com/en/3.1.x/templates/#comments this keeps the HTML out of the DOM.

Reply:
mb - will open a new PR